Resolutions/TekkenTagTournament2_Resolution/patches.txt

```diff
@@ -0,0 +1,5 @@
+[tekken32]
+moduleMatches = 0x465EA719
+0x10433450 = .float $width/$height
+0x101798a4 = .float (0.017453292 * (($width/$height)/($gameWidth/$gameHeight)))
+0x1072d978 = .float (0.017453292 * (($width/$height)/($gameWidth/$gameHeight)))
```

Comment:
Uh, could you maybe add some code comments on what each line does?

Reply:
Ok sure

Reply:
ehm i dont know how to change the files i already uploaded but the first one is culling and the second one is props culling

Reply:
i fix it